sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -55,6 +55,7 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
     TimeZone.setDefault(TimeZone.getTimeZone("America/Los_Angeles"))
     // Add Locale setting
     Locale.setDefault(Locale.US)
+    sql("DROP TEMPORARY FUNCTION IF EXISTS udtf_count2")
```

Comment:
ya. I think it is because `SQLQuerySuite` which creates `udtf_count2` but never drops it.